Formula/truffle.rb

```diff
@@ -0,0 +1,22 @@
+require "language/node"
+
+class Truffle < Formula
+  desc "Development environment, testing framework and asset pipeline for Ethereum"
+  homepage "https://trufflesuite.com"
+  url "https://registry.npmjs.org/truffle/-/truffle-5.1.63.tgz"
+  sha256 "80425e794393b2b8a61b915d6f8389c5bc443524c9108a4f985269d1edec1612"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system bin/"truffle", "init"
+    system bin/"truffle", "compile"
+    system bin/"truffle", "test"
```

Comment:
Can we check any output here?

Reply:
The output is along the lines of "Compilation successful" or "Tests passed", and there is a non-zero exit code if those steps fail anyways, so I'm not sure if checking the output is particularly helpful.

Reply:
> does system trap/fail on non-zero exit codes?

Yes.